extensions/gitlab/package.json

```
@@ -188,6 +188,32 @@
       "default": false,
       "title": "Behavior",
       "description": "Pop to root after e.g. open the browser"
+    },
+    {
+      "name": "ignorecerts",
+      "type": "checkbox",
+      "label": "Ignore Certificate Errors (not recommended)",
+      "required": false,
+      "default": false,
+      "title": "Certificate errors",
```

Nitpicking, but to keep it consistent it would be good to have this in title case
```suggestion
      "title": "Certificate Errors",
```